Assets/Scripts/UI/Tooltip/NodeUITooltipTrigger.cs

```diff
@@ -4,17 +4,18 @@ public class NodeUITooltipTrigger : TooltipTrigger {
     private TowerInfo currentTower;
     private TowerInfo newTower;
     private bool isUpgrade;
-    public bool isNotAvailable { get; set; }
+    public bool isFix = false;

```

Comment:
```suggestion
    private bool isFix = false;

```

instead of making this public, do similar to `isUpgrade` and infer from currentTower the value of  `isFix`. See my next comment below

Reply:
Paiseh, I think the naming of this variable is a bit wrong, it should be `isFixButton`. 

It is a flag for the `NodeUITooltipTrigger.cs` to format the content for `fixbutton` only.

![image](https://user-images.githubusercontent.com/62177572/192528949-b9e0de62-3878-4fe7-baea-a54ef4723b1d.png)